### PR TITLE
Feature/Add sse module documentation

### PR DESCRIPTION
**Pull request by sebbi08**

### CODE REVIEW

**Review by jkoenig134 — CHANGES_REQUESTED** [review]
You missed the modules page, we have every module on two separate pages.

File: _docs_operate/modules.md [hunk 1]
```diff
@@ -87,6 +87,11 @@ For this, the Connector supports the configuration of webhooks which are called
 
 Keep in mind that you need to synchronize the state of the Connector with the Backbone in order to receive events. The `sync` Module automates this, but you can also do this manually by calling the `/api/v2/Account/Sync` route.
 
+### Sever sent events <a href="{% link _docs_operate/configuration.md %}#sse"><i class="fas fa-fw fa-cog"/></a> {#sse}
+
+The server sent events module allows the connector to open an one way connection from the backbone to the connector.
```
Comment (britsta) [inline _docs_operate/modules.md:92]: Capitalize the words "Module", "Connector" and "Backbone" as it is already done in the rest of the text on this page. Same for the sentence below.

File: _docs_operate/modules.md [hunk 2]
```diff
@@ -87,6 +87,11 @@ For this, the Connector supports the configuration of webhooks which are called
...
+This allows the backbone to communicate needed syncs to the connector and ditch the sync module that polls every x seconds.
```
Comment (britsta) [inline _docs_operate/modules.md:93]: I'd rephrase: "...and ditch**es** the `sync` Module, which polls changes from the Backbone at regular time intervals.".

File: _docs_operate/configuration.md [hunk 3]
```diff
@@ -607,6 +607,30 @@ This module is deprecated in favor of the [Message Broker Publisher](#messagebro
...
+}
+```
+
+- **enabled** `default: false`
+
+  Enable or disable the sse Module.
+
+- **baseUrlOverride** `default: ''`
+
+  The sse module by default uses the backbone base url. This option allows it to be overwritten.
```
Comment (britsta) [inline _docs_operate/configuration.md:632]: Capitalize the words "Module" and "Backbone" as it is already done in the rest of the text on this page.

File: _docs_operate/configuration.md [hunk 4]
```diff
@@ -607,6 +607,30 @@ This module is deprecated in favor of the [Message Broker Publisher](#messagebro
 
 You can find type definitions of the event data in the [Connector Events]({% link _docs_integrate/connector-events.md %}) section.
 
+#### server sent event <a href="{% link _docs_operate/modules.md %}#sse"><i class="fas fa-fw fa-info-circle"/></a> {#sse}
```
Comment (britsta) [inline _docs_operate/configuration.md:610]: I would make sure that "sse" is used in the header, for example: "sse for server sent events"

**Comment by sebbi08** [comment]
> You missed the modules page, we have every module on two separate pages.

Do you mean another file than `_docs_operate/modules.md` ? That I already added.

File: _docs_operate/modules.md (hunk 2 above)
Comment (sebbi08) [reply _docs_operate/modules.md]: I completely rephrased it into two sentences.

File: _docs_operate/modules.md [hunk 5]
```diff
@@ -87,6 +87,12 @@ For this, the Connector supports the configuration of webhooks which are called
 
 Keep in mind that you need to synchronize the state of the Connector with the Backbone in order to receive events. The `sync` Module automates this, but you can also do this manually by calling the `/api/v2/Account/Sync` route.
 
+### sse (sever sent events) <a href="{% link _docs_operate/configuration.md %}#sse"><i class="fas fa-fw fa-cog"/></a> {#sse}
+
+The sse Module allows the Connector to open an one way connection from the Backbone to the Connector.
```
Comment (Milena-Czierlinski) [inline _docs_operate/modules.md:92]: a one way (even though a is followed by a vocal ;) )

File: _docs_operate/modules.md [hunk 6]
```diff
@@ -87,6 +87,12 @@ For this, the Connector supports the configuration of webhooks which are called
...
+This allows the Backbone to communicate needed syncs to the Connector.
+For this the sync Module is not needed anymore and can be deactivated.
```
Comment (Milena-Czierlinski) [inline _docs_operate/modules.md:94]: Therefore the sync Module is not needed anymore and can be deactivated.

File: _docs_operate/configuration.md [hunk 7]
```diff
@@ -607,6 +607,30 @@ This module is deprecated in favor of the [Message Broker Publisher](#messagebro
...
+
+**Sample Configuration:**
+
+```jsonc
+{
+  // ...
+
+  "modules": {
+    "sse": {
+      "baseUrlOverride": "https://example.com"
```
Comment (Milena-Czierlinski) [inline _docs_operate/configuration.md:620]: Does it make sense to override the base url, if the module is disabled?

Comment (jkoenig134) [inline _docs_operate/configuration.md:620]: please don't document this. This is only for development purposes and should not be used.

Comment (jkoenig134) [inline _docs_operate/configuration.md:620]: enabled is missing in this example

File: _docs_operate/modules.md (hunk 5 above)
Comment (jkoenig134) [inline _docs_operate/modules.md:92]: IMO the whole description of the module is way to technical. And the technical details are wrong. The Backbone isn't the actor that opens a connection to the Connector - it is the other way round. As someone who reads this I would directly see this module as unusable as I don't want to expose my connector to the internet.

File: _docs_operate/configuration.md [hunk 8]
```diff
@@ -607,6 +607,30 @@ This module is deprecated in favor of the [Message Broker Publisher](#messagebro
 
 You can find type definitions of the event data in the [Connector Events]({% link _docs_integrate/connector-events.md %}) section.
 
+#### sse (sever sent events) <a href="{% link _docs_operate/modules.md %}#sse"><i class="fas fa-fw fa-info-circle"/></a> {#sse}
+
+**Sample Configuration:**
```
Comment (jkoenig134) [inline _docs_operate/configuration.md:612]: please add a warning that sse needs additional configuration on the backbone and may not be available on all backbone instances.

File: _docs_operate/modules.md [hunk 9]
```diff
@@ -87,6 +87,12 @@ For this, the Connector supports the configuration of webhooks which are called
 
 Keep in mind that you need to synchronize the state of the Connector with the Backbone in order to receive events. The `sync` Module automates this, but you can also do this manually by calling the `/api/v2/Account/Sync` route.
 
+### sse (sever sent events) <a href="{% link _docs_operate/configuration.md %}#sse"><i class="fas fa-fw fa-cog"/></a> {#sse}
+
+The sse Module allows the Connector to receive events from the Backbone without exposing the Connector to the Internet.
+This allows the Backbone to communicate required syncs to the connector.
+The sync module is therefore no longer required and can be disabled.
```
Comment (jkoenig134) [inline _docs_operate/modules.md:94]: we should make this required (not allow having sync and sse enabled together)

File: _docs_operate/modules.md [hunk 10]
```diff
@@ -87,6 +87,12 @@ For this, the Connector supports the configuration of webhooks which are called
 
 Keep in mind that you need to synchronize the state of the Connector with the Backbone in order to receive events. The `sync` Module automates this, but you can also do this manually by calling the `/api/v2/Account/Sync` route.
 
+### sse (sever sent events) <a href="{% link _docs_operate/configuration.md %}#sse"><i class="fas fa-fw fa-cog"/></a> {#sse}
+
+The sse Module allows the Connector to receive events from the Backbone without exposing the Connector to the Internet.
+This allows the Backbone to communicate required syncs to the connector.
```
Comment (jkoenig134) [inline _docs_operate/modules.md:93]: IMO this wording is too technical. Wtb: `The Connector reacts to these events and ensures automatic synchronisation.`

File: _docs_operate/modules.md (hunk 9 above)
Comment (sebbi08) [reply _docs_operate/modules.md]: Already dopcumented i will open a PR for the connector

File: _docs_operate/modules.md [hunk 11]
```diff
@@ -67,6 +67,9 @@ Compared to [webhooks](#webhooks), this gives you the full feature set of these
 
 The `sync` Module regularly fetches changes from the Backbone (e.g. new Messages / new incoming Relationship Requests). This process automatically triggers the events used by other Modules like the `webhooks` Module.
 
+The sync module and the sse module are not compatible. The sync module will be disabled if both are active.
```
Comment (Magnus-Kuhn) [inline _docs_operate/modules.md:70]: maybe add a link to the sse module?

**Review by jkoenig134 — APPROVED** [review]

File: _docs_operate/modules.md (hunk 11 above)
Comment (jkoenig134) [reply _docs_operate/modules.md:70]: yep, should be useful.

File: _docs_operate/modules.md [hunk 12]
```diff
@@ -65,7 +65,10 @@ Compared to [webhooks](#webhooks), this gives you the full feature set of these
 
 ### Sync <a href="{% link _docs_operate/configuration.md %}#sync"><i class="fas fa-fw fa-cog"/></a> {#sync}
 
-The `sync` Module regularly fetches changes from the Backbone (e.g. new Messages / new incoming Relationship Requests). This process automatically triggers the events used by other Modules like the `webhooks` Module.
+The `sync` Module regularly fetches changes from the Backbone (e.g. new Messages / new incoming Relationship Requests). This process automatically triggers the events used by other Modules like the [webhooks Module](#webhooks).
+
+The sync module and the [sse module](#sse) are not compatible. The sync module will be disabled if both are active.
```
Comment (jkoenig134) [inline _docs_operate/modules.md:70]:
```suggestion
The sync Module and the [sse Module](#sse) are not compatible. The sync Module will be disabled if both are active.
```

**Review by jkoenig134 — APPROVED** [review]
